doc/source/tutorial/interpolate/1D.rst

```diff
@@ -339,6 +341,55 @@ same data as in the previous example:
 ..             `next`.
 
 
+.. _tutorial-interpolate_interp1d_replacements:
+
+Recommended replacements for `interp1d` modes
+---------------------------------------------
+
+As mentioned, `interp1d` class is *legacy*: we have no plans to remove it; we
+are going to keep supporting its existing usages; however
+we believe there are better alternatives which we recommend using in new code.
+
+Here we list specific recommendations, depending on the interpolation ``kind``.
+
+Linear interpolation, ``kind="linear"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default recommendation is to use `numpy.interp` function. Alternatively,
+you can use linear splines, ``make_interp_spline(x, y, k=1)``, see
+:ref:`this section for a discussion<tutorial-interpolate_linear_spline>`.
+
+
+Spline interpolators, ``kind="quadatic"`` or ``"cubic"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Under the hood, ``interp1d`` delegates to ``make_interp_spline``, so we recommend
```

Comment:
```suggestion
Under the hood, `interp1d` delegates to `make_interp_spline`, so we recommend
```

Reply:
Thanks, indeed. Fixed and force-pushed, both this and the other suggestion.